examples/resources/qdrant-cloud_accounts_role/resource.tf

```diff
@@ -0,0 +1,49 @@
+terraform {
+  required_version = ">= 1.7.0"
+  required_providers {
+    qdrant-cloud = {
+      source  = "qdrant/qdrant-cloud"
+      version = ">=1.1.0"
+    }
+  }
+}
+
+// Provider configuration
+provider "qdrant-cloud" {
+  api_key    = "" // API Key generated in Qdrant Cloud (required)
+  account_id = "" // Default account ID (can be overridden per resource)
+}
+
+// Create a custom IAM Role
+resource "qdrant-cloud_accounts_role" "backup_operator" {
+  name        = "backup-operator"
+  description = "Can create and restore cluster backups."
+
+  // Assign permissions for this role
+  permissions {
+    value    = "read:backups"
+    category = "Cluster"
```

Comment:
IMO category is a read-only setting, so shouldn't be required here.

Reply:
done.